client/src/pages/Profile.jsx

```
@@ -1,5 +1,117 @@
-    <div>Profile</div>
-  )
+    <div className={styles.wrapper}>
+      <input
+        type="file"
+        accept="image/*"
+        ref={imageFileRef}
+        onChange={onChangeImageFile}
+        style={{ display: 'none' }}
+        id="input"
+      />
+      <img src={previewUrl || initialData.profileImageUrl} className={styles.profileImage} />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)